services/provider/server/server.go

```diff
@@ -465,7 +466,9 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 		// The current proposal is to read the clientProfile name from the storageConsumer status and
 		// the remote ClientProfile name should be fetched from the GetClientsInfo rpc
 		clientName := consumerResource.Status.Client.Name
-		clientProfileName := util.CalculateMD5Hash(fmt.Sprintf("%s-ceph-rbd", clientName))
+		// Using the same hash function we have in client op to calculate the clientProfile Name
+		clientProfileHash := md5.Sum([]byte(fmt.Sprintf("%s-ceph-rbd", clientName)))
+		clientProfileName := hex.EncodeToString(clientProfileHash[:])
```

Comment:
so in provider before hashing we are turning them into a slice 😅

Reply:
Yes right